uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Import/Helpers/DataImportTrait.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
+use CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChange as LeaveBalanceChange;
+use CRM_HRLeaveAndAbsences_Service_LeaveRequestComment as LeaveRequestCommentService;
+use CRM_HRLeaveAndAbsences_BAO_AbsenceType as AbsenceType;
+
+trait CRM_HRLeaveAndAbsences_Import_Helpers_DataImportTrait {
```

Comment:
The trait has three or so methods that helps create leave request, balance changes and comments. Didn't want to clutter the parser file with those methods and only want the validation methods only in the parser file. What do you suggest?